test/e2e/e2e_leaderelection_test.go

```diff
@@ -35,13 +35,46 @@ import (
 	"sigs.k8s.io/descheduler/cmd/descheduler/app/options"
 	"sigs.k8s.io/descheduler/pkg/api"
 	"sigs.k8s.io/descheduler/pkg/descheduler"
+	"sigs.k8s.io/descheduler/pkg/framework/plugins/defaultevictor"
+	"sigs.k8s.io/descheduler/pkg/framework/plugins/podlifetime"
 )
 
 // Should use something like "sigs.k8s.io/descheduler/pkg/framework/plugins/removepodshavingtoomanyrestarts"
-func leaderElectionPolicy(targetNamespace string) *api.DeschedulerPolicy {
+func PodLifeTimePolicy(targetNamespace string, maxPodLifeTimeSeconds *uint) *api.DeschedulerPolicy {
 	return &api.DeschedulerPolicy{
 		Profiles: []api.DeschedulerProfile{
-			{},
+			{
+				Name: "PodLifeTimeProfile",
+				PluginConfigs: []api.PluginConfig{
+					{
+						Name: podlifetime.PluginName,
+						Args: &podlifetime.PodLifeTimeArgs{
+							MaxPodLifeTimeSeconds: maxPodLifeTimeSeconds, // [TODO] The values to set
+							Namespaces: &api.Namespaces{
+								Include: []string{targetNamespace},
+							},
+						},
+					},
+					{
+						Name: defaultevictor.PluginName,
+						Args: &defaultevictor.DefaultEvictorArgs{
```

Comment:
Args in &defaultevictor.DefaultEvictorArgs follows #1472 
Which may not be the proper args here